packages/node-cli/src/validation/InteractiveAnswersValidator.js

```diff
@@ -149,4 +151,16 @@ module.exports = {
 			? VALIDATION_RESULT_SUCCESS
 			: VALIDATION_RESULT_FAILURE(NodeTranslationService.getMessage(ANSWERS_VALIDATION_MESSAGES.MAX_LENGTH, maxLength));
 	},
+
+	validateNonProductionDomain(fieldValue) {
+		return !fieldValue.match(PRODUCTION_DOMAIN_REGEX)
+			? VALIDATION_RESULT_SUCCESS
+			: VALIDATION_RESULT_FAILURE(NodeTranslationService.getMessage(ANSWERS_VALIDATION_MESSAGES.PRODUCTION_DOMAIN));
```

Comment:
I would suggest trying to use positive conditional instead of negative. It's always harder reading code with negative ones.

Reply:
I thought the same but @davidecorreu suggested to do it like this. I'm fine with either way.

Reply:
What do you think?

Reply:
I see! As a general rule I always try to promote positive conditional, sometimes looking from a different perspective helps to apply the rule:
* Why not calling the function validateDomain?

In any case this comment is not a blocking one so you can go ahead and decide offline 👍.

Reply:
BTW good code 😃! You are the real Full Stack

Reply:
Thanks, Victor!!! Getting there 💪